<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customDialog</class>
 <widget class="QDialog" name="customDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;High:     </string>
       </property>
       <property name="buddy">
        <cstring>highLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="highLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Width:    </string>
       </property>
       <property name="buddy">
        <cstring>widthLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="widthLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;MineCount:</string>
       </property>
       <property name="buddy">
        <cstring>mineLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mineLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="yesPushButton">
       <property name="text">
        <string>&amp;Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="noPushButton">
       <property name="text">
        <string>&amp;No</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>yesPushButton</sender>
   <signal>clicked()</signal>
   <receiver>customDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noPushButton</sender>
   <signal>clicked()</signal>
   <receiver>customDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
